src/parser/Configuration.ts

```diff
@@ -69,7 +82,23 @@ export class Configuration {
         this.outputPath = outputPath;
         this.parseMetrics = true;
         this.parseDependencies = parseDependencies;
-        this.exclusions = new Set(exclusions.split(",").map((exclusion) => exclusion.trim()));
+
+        if (exclusions.length > 0) {
+            this.exclusions = new Set(exclusions.split(",").map((exclusion) => exclusion.trim()));
+        } else {
+            this.exclusions = new Set();
+        }
```

Comment:
You could use a ternary here.

Reply:
I could. I prefer to not write "one line that does it all". But I can change that here.